samples/algorithms/Shor.qs

```diff
@@ -70,69 +85,67 @@ namespace Microsoft.Quantum.Samples.Shor {
         }
         until foundFactors
         fixup {
-            Message("The estimated period did not yield a valid factor, trying again.");
+            Message("The estimated period did not yield a valid factor. " + 
+                "Trying again.");
         }
 
         // Return the factorization
         return factors;
     }
 
     /// # Summary
-    /// Tries to find the factors of `modulus`
-    /// given a `period` and `generator`.
+    /// Tries to find the factors of `modulus` given a `period` and `generator`.
```

Comment:
For some of these spots that put the comments back on one line, I remember Dmitry had purposefully adjusted the formatting to make it more likely to fit on screen without horizontal scroll bars. Is that no longer a concern? I can see those type of edits here and in Grover.qs, but for some reason the other files show up in the diff as deleted and then added instead of moved, so I can't see exactly what changes were made.

Reply:
I made adjustments to the length of lines but made sure to keep lines column count below 80, which I think it's the standard. I've also been verifying how it looks in the playground and everything fits nicely.